lib/init.sh

```diff
@@ -44,23 +44,32 @@ parse_cmdline()
     # ... parameters with '=' go into init's environment ...
     for _param in $cmdline; do case $_param in
         # pass to init
-	--)           init_args+=${cmdline#*--}; break ;;
-	quiet)        init_args+=" --quiet" ;;
+        --)           init_args="${init_args}${cmdline#*--}"; break ;;
+        quiet)        init_args="${init_args} --quiet" ;;
 
+        # internal functions
         rdpanic) trap - EXIT ;;
         rddebug) set -x ;;
-	break) break_init=1 ;;
+        break) break_init=1 ;;
```

Comment:
`debug` may be a more common name. Rename `rddebug` to `verbose`, and `break` to `debug`?

We should also document these options (could in another PR)